.travis.yml

```diff
@@ -6,6 +6,7 @@ env:
 script:
 - bundle exec rake db:create
 - bundle exec rake db:test:prepare
+- bundle exec rake db:seed RAILS_ENV=test
```

Comment:
Ok! Could you please drop a `TODO` comment here, in order to remind us to find a workaround for this issue? :)